gecko/glue/include/nsComponentManager.h

```diff
@@ -9,7 +9,12 @@
 
 #include "nsXPCOM.h"
 
+#if defined(DARWIN)
```

Comment:
We should use XP_MACOS for Mozilla code.

Reply:
PR updated!